input/Code/GetTimesSeries.R

```diff
@@ -30,6 +30,7 @@ lapply(neededPackages, function(x) suppressPackageStartupMessages(
 # Change your API (from renewables ninja account)
 # insert your API authorisation token here
 token = 'efde8da5c66ef97b495ddddf01ef55da4c8a04e6'
```

Comment:
are you sure it is a good idea to expose a private API key in a public repository?